cumulus/client/pov-recovery/src/tests.rs

```diff
@@ -198,11 +199,15 @@ impl<Block: BlockT> BlockBackend<Block> for ParachainClient<Block> {
 		unimplemented!()
 	}
 
-	fn indexed_transaction(&self, _: Block::Hash) -> sp_blockchain::Result<Option<Vec<u8>>> {
+	fn indexed_transaction(&self, _: H256) -> sp_blockchain::Result<Option<Vec<u8>>> {
```

Comment:
dq: Does the `Block::Hash` resolve to something different here? Or we should update the other methods of the trait?